src/server/utilities/cspHeader/index.js

```diff
@@ -127,6 +133,11 @@ const directives = {
       'https://www.bbc.co.uk', // STY include
       'http://www.bbc.co.uk', // for localhost STY include
       'https://bbc-maps.carto.com', // STY include maps
+      // Ads
+      'https://bcp.crwdcntrl.net',
+      'https://edigitalsurvey.com',
```

Comment:
Suggesting this change since these lines might be reorganised in future
```suggestion
      'https://bcp.crwdcntrl.net', // ads
      'https://edigitalsurvey.com', // ads
```